### PR TITLE
net/relayd - sync with OpenBSD 

### PR DESCRIPTION
**Pull request by AdSchellevis**
An updated relayd version is on it's way, which should bring FreeBSD's version in sync with upstream OpenBSD (https://github.com/freebsd/freebsd-ports/commit/c9ba90c07ddc5b9f4232f831cf301fdc939303f3). Since some small things have changed, we need a migration and a some small config changes.

A test build is available  at https://pkg.opnsense.org/FreeBSD:13:amd64/snapshots/misc/relayd-7.3.2023.05.09.p2.pkg

### CODE REVIEW

File: net/relayd/src/opnsense/service/templates/OPNsense/Relayd/relayd.conf
Comment (fbrendel) [inline net/relayd/src/opnsense/service/templates/OPNsense/Relayd/relayd.conf:1]: Hi @AdSchellevis 
the `check send` _ssl_ option has changed to _tls_ too.
``` diff
diff --git a/net/relayd/src/opnsense/service/templates/OPNsense/Relayd/relayd.conf b/net/relayd/src/opnsense/service/templates/OPNsense/Relayd/relayd.conf
index 16f73445..bd25e981 100644
--- a/net/relayd/src/opnsense/service/templates/OPNsense/Relayd/relayd.conf
+++ b/net/relayd/src/opnsense/service/templates/OPNsense/Relayd/relayd.conf
@@ -110,7 +110,7 @@ disable
 {%          elif tablecheck.type == 'send' and tablecheck.expect is defined %}
 {%            set _tablecheck = 'check ' ~ tablecheck.type ~ ' "' ~ tablecheck.data ~ '" expect "' ~ tablecheck.expect ~ '"' %}
 {%            if tablecheck.ssl|default('0') == '1' %}
-{%              set _tablecheck = _tablecheck ~ ' ssl' %}
+{%              set _tablecheck = _tablecheck ~ ' tls' %}
 {%            endif %}
 {%          else %}
 {%            set _tablecheck = 'check ' ~ tablecheck.type %}
@@ -145,7 +145,7 @@ disable
 {%          elif backuptablecheck.type == 'send' and backuptablecheck.expect is defined %}
 {%            set _backuptablecheck = 'check ' ~ backuptablecheck.type ~ ' "' ~ backuptablecheck.data ~ '" expect "' ~ backuptablecheck.expect ~ '"' %}
 {%            if backuptablecheck.ssl|default('0') == '1' %}
-{%              set _backuptablecheck = _backuptablecheck ~ ' ssl' %}
+{%              set _backuptablecheck = _backuptablecheck ~ ' tls' %}
 {%            endif %}
 {%          else %}
 {%            set _backuptablecheck = 'check ' ~ backuptablecheck.type %}
```

**Review by fbrendel — CHANGES_REQUESTED** [review]

File: net/relayd/src/opnsense/service/templates/OPNsense/Relayd/relayd.conf
Comment (AdSchellevis) [reply net/relayd/src/opnsense/service/templates/OPNsense/Relayd/relayd.conf:1]: @fbrendel nice catch! thanks!

**Review by fbrendel — APPROVED** [review]
Looks good. 
Thank you :smiley:

**Comment by fichtner** [comment]
@AdSchellevis Feel free to merge as the build for 23.7.1 is already done

**Comment by AdSchellevis** [comment]
@fbrendel thanks for reviewing!

**Pull request merged**